Homework5: Aliaksei Uhalnikau

alexugalek opened the pull request.

alexugalek committed:
Add Homework5 tasks

alexugalek committed:
Comment all imports

belovmd reviewed (changes requested):
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework5.md):
- 13
- 10
- 5
- 1

alexugalek committed:
Fix all from review list
 - Add histograms
 - Remove redundant comments
 - Add time parametr in decorator_example
 - Use isinstance(types.FunctionType) condition in runner_func

alexugalek committed:
Fix flake-8

alexugalek committed:
Edit gistograms

belovmd reviewed (commented):
10
